lib/browse_everything.rb

```diff
@@ -40,6 +40,7 @@ module Google
   end
 
   class InitializationError < RuntimeError; end
+  class ConfigurationError < StandardError; end
```

Comment:
I'm tempted to suggest this should inherit `Errno::ENOENT`, instead of `StandardError`.

If I handle the error emitted by current versions, this will break my handling.